stl/inc/xutility

```diff
@@ -252,6 +252,25 @@ __declspec(noalias) size_t __stdcall __std_mismatch_8(const void* _First1, const
 
 } // extern "C"
 
+#if _HAS_CXX20 && !defined(__clang__) && !defined(__EDG__) // TRANSITION, DevCom-10798069
+_STD_BEGIN
+struct _Secret_placement_new_tag {
+    explicit _Secret_placement_new_tag() = default;
+};
+_STD_END
+
+template <_STD same_as<_STD _Secret_placement_new_tag> _Tag>
+_NODISCARD _Ret_notnull_ _Post_writable_byte_size_(_Size)
+    _Post_satisfies_(return == _Where) constexpr void* __CRTDECL operator new[](
+        size_t _Size, _Tag, _Writable_bytes_(_Size) void* _Where) noexcept {
+    (void) _Size;
+    return _Where;
+}
+
+template <_STD same_as<_STD _Secret_placement_new_tag> _Tag>
+constexpr void __CRTDECL operator delete[](void*, _Tag, void*) noexcept {}
+#endif // _HAS_CXX20 && !defined(__clang__) && !defined(__EDG__)

```

Comment:
The code above is "workaround", right? We should try to use "workaround" / "no workaround" comments in new code, which makes it a lot easier to remember what to do when bugs are fixed.